<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNewTask_Form</class>
 <widget class="QWidget" name="AddNewTask_Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add New Task</string>
  </property>
  <widget class="QListWidget" name="WorkType_listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>256</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>.Work</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="WorkType_lineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>.Work Type</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="TaskName_lineEdit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>100</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>80</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>.Task Name</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="TaskInstruction_textEdit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>150</y>
     <width>261</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>130</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>.My Task Instruction</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>440</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>263</x>
     <y>30</y>
     <width>21</width>
     <height>401</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="Create_BT">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>440</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>531</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>AddNewTask_Form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>493</x>
     <y>450</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>451</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
